week1/neumorphic.css

```diff
@@ -1 +1,59 @@
-/* todo */
\ No newline at end of file
+*,
+h1,
+input[type="text"] {
+    font-family: helvetica, sans-serif;
+}
+
+h1 {
+    font-weight: lighter;
+    font-size: 40px;
+    font-variant-caps: all-petite-caps;
+}
+
+main {
+    border-radius: 25px;
+    -webkit-box-shadow: 10px 12px 37px -7px rgba(0, 0, 0, 0.49);
+    -moz-box-shadow: 10px 12px 37px -7px rgba(0, 0, 0, 0.49);
+    box-shadow: 10px 12px 37px -7px rgba(0, 0, 0, 0.49);
```

Comment:
I would assume that box-shadow is now well-enough supported in all browsers that we do not need to repeat in -webkit -moz. But if you do, better use a css custom property (var) to remove the repetition

Reply:
yes, this is already outdated and was added in a later version (I am assuming you went through commit by commit)